Week3/More JavaScript.js

```diff
@@ -0,0 +1,180 @@
+
+/**************************** More JavaScript ***************************** */
+
+// 1. Create a function that takes 3 arguments and returns the sum of the these arguments.
+
+function f1(a,b,c) {
```

Comment:
Nice! But could you give the function a more descriptive name? `f1` doesn't really say much about what the function does.

Reply:
I changed f1 with sum. I think it is more suitable